html/browsers/windows/noreferrer.html

```diff
@@ -0,0 +1,25 @@
+<!doctype html>
+<title>rel=noreferrer nullifies window.opener</title>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<div id="log"></div>
+<script>
+  async_test(function(t) {
+    localStorage.clear()
```

Comment:
I asked about that earlier and I think @Ms2ger replied that it wasn't much of an issue?